flamingpy/cv/ops.py

```diff
@@ -392,3 +392,10 @@ def draw(self, **kwargs):
         cv_opts = {"color_nodes": "state", "state_colors": {"GKP": "gold", "p": "blue"}}
         updated_opts = {**cv_opts, **kwargs}
         return self.egraph.draw(**updated_opts)
+
+    def draw_adjacency(self, **kwargs):
```

Comment:
Instead of having a draw_adjacency function (which can be called from the EGraph) let's have a draw_SCZ)